saleor/graphql/app/types.py

```diff
@@ -124,6 +125,26 @@ class Meta:
         node = AppExtension
 
 
+class AppManifestWebhook(graphene.ObjectType):
+    name = graphene.String(required=True)
+    events = NonNullList(WebhookEventTypeEnum, required=True)
+    query = graphene.String(required=True)
+    target_url = graphene.String(required=True)
+    is_active = graphene.Boolean(required=True)
```

Comment:
Could we add the descriptions to the fields here?

Reply:
@maarcingebala 
* how would I know if that's preview feature? who can provide answer to that question?
* assuming, that I'm providing the whole `webhooks` as `NonNullList` in this version, should I mark `ADDED_IN_34` just that list? Should I mark also nested fields? Should I mark just nested fields, leaving list unmarked?

Reply:
- Usually, an owner of the feature should be able to provide the answer; I don't know who owns this feature, maybe @timuric?
- Since we are adding an entirely new type here, we could add ADDED_IN_34 at the type description level. To add a description to the type you can add:
```
class Meta:
    description = ""
```